Gemfile

```diff
@@ -55,6 +55,8 @@ gem 'html2haml'
 
 gem 'devise', '~> 4.9'
 
+gem 'kaminari'
```

Comment:
Let's add the gem's version you are using, so we don't get unexepcted updates later!